examples/ner/run_ner.py

```diff
@@ -0,0 +1,183 @@
+        default=16, metadata={"help": "Batch size used for training."}
+    )
+    learning_rate: float = field(
+        default=5e-05, metadata={"help": "Learning rate OneCycleLR scheduler"}
+    )
+    seed: int = field(
+        default=42, metadata={"help": "Seed used for reproducible fine-tuning results."}
+    )
+    device: str = field(default="cuda:0", metadata={"help": "CUDA device string."})
+    weight_decay: float = field(
+        default=0.0, metadata={"help": "Weight decay for optimizer."}
+    )
+    embeddings_storage_mode: str = field(
+        default="none", metadata={"help": "Defines embedding storage method."}
+    )
+
+
+@dataclass
+class FlertArguments:
+    context_size: int = field(
+        default=0, metadata={"help": "Context size when using FLERT approach."}
+    )
+    respect_document_boundaries: bool = field(
+        default=False,
+        metadata={
+            "help": "Whether to respect document boundaries or not when using FLERT."
+        },
+    )
+
+
+@dataclass
+class DataArguments:
+    dataset_name: str = field(metadata={"help": "Flair NER dataset name."})
+    dataset_arguments: str = field(
+        default="", metadata={"help": "Dataset arguments for Flair NER dataset."}
+    )
+    output_dir: str = field(
+        default="resources/taggers/ner",
+        metadata={"help": "Defines output directory for final fine-tuned model."},
+    )
+
+
+def get_flair_corpus(data_args):
+    ner_task_mapping = {}
+
+    for name, obj in inspect.getmembers(flair.datasets.sequence_labeling):
+        if inspect.isclass(obj):
+            if (
+                name.startswith("NER")
+                or name.startswith("CONLL")
+                or name.startswith("WNUT")
+            ):
+                ner_task_mapping[name] = obj
+
+    dataset_args = {}
+    dataset_name = data_args.dataset_name
+
+    if data_args.dataset_arguments:
+        dataset_args = json.loads(data_args.dataset_arguments)
+
+    if not dataset_name in ner_task_mapping:
+        raise ValueError(
+            f"Dataset name {dataset_name} is not a valid Flair datasets name!"
+        )
+
+    return ner_task_mapping[dataset_name](**dataset_args)
+
+
+def main():
+    parser = HfArgumentParser(
+        (ModelArguments, TrainingArguments, FlertArguments, DataArguments)
+    )
+
+    if len(sys.argv) == 2 and sys.argv[1].endswith(".json"):
+        (
+            model_args,
+            training_args,
+            flert_args,
+            data_args,
+        ) = parser.parse_json_file(json_file=os.path.abspath(sys.argv[1]))
+    else:
+        (
+            model_args,
+            training_args,
+            flert_args,
+            data_args,
+        ) = parser.parse_args_into_dataclasses()
+
+    set_seed(training_args.seed)
+
+    flair.device = training_args.device
+
+    corpus = get_flair_corpus(data_args)
+
+    logger.info(corpus)
+
+    tag_type: str = "ner"
+    tag_dictionary = corpus.make_tag_dictionary(tag_type=tag_type)
```

Comment:
Oh, it seems that this method is outdated:

```bash
run_ner.py:143: DeprecationWarning: Call to deprecated method make_tag_dictionary. (Use 'make_label_dictionary' instead.) -- Deprecated since version 0.8.
```